components/tools/OmeroJava/test/integration/ManagedRepositoryTest.java

```diff
@@ -95,7 +95,7 @@
 
     @BeforeMethod
     public void setupNewUser() throws Exception {
-        newUserAndGroup("rw----");
+        //newUserAndGroup("rw----");
```

Comment:
I assume this is a temporary commenting out?

Reply:
sorry that was my initial commit while testing
@mtbc could you remove the line?
This is not needed

Reply:
Sure, though I'll hold off for now as you may also have other commits arising from comments.